api/tests/unit/domain/usecases/authenticate-user_test.js

```diff
@@ -351,4 +355,156 @@ describe('Unit | Application | UseCase | authenticate-user', function () {
       expect(error.meta).to.equal('RESET_PASSWORD_TOKEN');
     });
   });
+
+  context('when user has a locale', function () {
+    it('does not update the user locale', async function () {
+      // given
+      const accessToken = 'jwt.access.token';
+      const source = 'pix';
+      const expirationDelaySeconds = 1;
+      const user = domainBuilder.buildUser({ email: userEmail });
```

Comment:
Pour être plus explicite :
```suggestion
      const user = domainBuilder.buildUser({ email: userEmail, locale: 'fr-FR' });
```